src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -1475,6 +1475,7 @@ private enum CallKind
         {
             Call,
             CallVirt,
+            ConstrainedCall,
```

Comment:
Curious. I was expecting this to be a `ConstrainedCallVirt`, and that CallVirt would know to not throw an exception on a null receiver for a static abstract.

Reply:
Must have been misremembering LDM conversations though.

---
In reply to: [613592402](https://github.com/dotnet/roslyn/pull/52605#discussion_r613592402) [](ancestors = 613592402)